test/parser_test.dart

```diff
@@ -329,14 +329,14 @@ main() {
       MAP_WITH_SOURCE_LOCATION,
       MAP_WITH_SOURCE_LOCATION_AND_NAME
     ]) {
-      var mapping = parseJson(expected);
+      var mapping = parseJson(expected) as SingleMapping;
       expect(mapping.toJson(), equals(expected));
 
-      mapping = parseJsonExtended(expected);
+      mapping = parseJsonExtended(expected) as SingleMapping;
       expect(mapping.toJson(), equals(expected));
     }
     // Invalid for this case
-    expect(() => parseJson(SOURCE_MAP_BUNDLE as dynamic), throws);
+    expect(() => parseJson(SOURCE_MAP_BUNDLE as dynamic), throws) as MappingBundle;
```

Comment:
this line makes no sense. Does this even compile?

Reply:
Sorry, copy+paste error. Should have added to the next line...